components/test_raftstore/src/cluster.rs

```diff
@@ -772,6 +772,16 @@ impl<T: Simulator> Cluster<T> {
                 sleep_ms(100);
                 continue;
             }
+            if resp
+                .get_header()
+                .get_error()
+                .get_message()
+                .contains("proposal dropped")
```

Comment:
Have you tried to verify if it's really fixing the issue? I can't grep the string in code, which probably means the fix is a noop.

Reply:
It's in `raft-rs`.